db/migrate/20170621202836_create_otp_requests_tracker.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOtpRequestsTracker < ActiveRecord::Migration
+  def change
+    create_table :otp_requests_trackers do |t|
+      t.text :encrypted_phone
+      t.timestamp :otp_last_sent_at
+      t.integer :otp_send_count, default: 0
+      t.string :attribute_cost
+      t.string :phone_fingerprint, default: '', null: false
+
+      t.index :phone_fingerprint, unique: true
+    end
```

Comment:
Can we add `t.timestamps` and an index on `updated_at`? That will let us have a periodic job to delete old records so the table doesn't grow too much